src/ImageSharp/Formats/Webp/WebpLookupTables.cs

```diff
@@ -1438,6 +1241,216 @@ static WebpLookupTables()
             InitializeFixedCostsI4();
         }
 
+        private static readonly byte[] Abs0Table =
```

Comment:
Make these use the ReadOnlySpan-trick?
Cf. https://github.com/SixLabors/ImageSharp/pull/1799#discussion_r740109460

It's already used in this class e.g. in `LogTable8Bit` (above).

Reply:
Before I make this change, i would like to ask for other opinions on that matter. Should we really go all in with that trick and change all `byte`,  `sbyte` arrays to ReadOnlySpan here?

Reply:
I would say yes.